### ISSUE TITLE
Missing jre folder in release build path

### ISSUE THREAD

**Issue by aberenguel** [sepinf-inc/IPED#1135]
I'm in master 30b973235 branch.

When I build the project, the directory `target/release/iped-4.0.0-snapshot/jre` is not being created.
This cause this exception when opening the case:
```
java.lang.NullPointerException: null
        at dpf.sp.gpinf.indexer.util.IOUtil.copyDirectory(IOUtil.java:315) ~[iped-utils-4.0.0-snapshot.jar:?]
        at dpf.sp.gpinf.indexer.process.Manager.prepareOutputFolder(Manager.java:819) ~[iped-engine-4.0.0-snapshot.jar:?]
        at dpf.sp.gpinf.indexer.process.Manager.process(Manager.java:221) ~[iped-engine-4.0.0-snapshot.jar:?]
        at dpf.sp.gpinf.indexer.Main.startManager(Main.java:147) [iped.jar:?]
        at dpf.sp.gpinf.indexer.Main.execute(Main.java:205) [iped.jar:?]
        at dpf.sp.gpinf.indexer.Main.main(Main.java:267) [iped.jar:?]
```

In addition, I think the `IOUtil.copyDirectory` could validate if the `origem` is a valid directory and throws a proper exception.

**Comment by aberenguel** [comment]
The maven log showed this line
```
[INFO] --- maven-dependency-plugin:2.10:unpack (unpack-jre) @ iped-app ---
[INFO] Configured Artifact: java:jre:11.0.13:zip
[INFO] jre-11.0.13.zip already unpacked.
```

I changed the `<overWrite>false</overWrite>` to true in `unpack-jre` execution, then it worked.

This is not the ideal. So I saw some things in https://maven.apache.org/plugins/maven-dependency-plugin/usage.html#Overwrite_Rules

The problem was my mistake. I cleaned the `./target` folder, but there was a file indicating that the package was unpacked in `./iped-app/target/dependency-maven-plugin-markers/java-jre-zip-11.0.13.marker`
